support/testing/address.go

```diff
@@ -1,46 +1,37 @@
 package testing
 
 import (
-	"math/rand"
-	"testing"
-
 	addr "github.com/filecoin-project/go-address"
+	"github.com/stretchr/testify/require"
+	"math/rand"
 )
 
-func NewIDAddr(t testing.TB, id uint64) addr.Address {
+func NewIDAddr(t require.TestingT, id uint64) addr.Address {
```

Comment:
Can we just use `*testing.T`? It's not clear why we can't do this the normal way.

Reply:
I started off with the assumption this would be run from the command line outside the test runner. This is a trivial interface to implement. That assumption was bad, though, so I'll revert.